src/devices/moes.ts

```diff
@@ -122,7 +118,7 @@ const definitions: DefinitionWithExtend[] = [
             legacy.tz.moes_thermostat_deadzone_temperature,
             legacy.tz.moes_thermostat_max_temperature_limit,
             legacy.tz.moes_thermostat_min_temperature_limit,
-            legacy.tz.moes_thermostat_program_schedule,
+            legacy.tz.moes_thermostat_program_schedule_v2,
```

Comment:
Let's not call it v2 but update the existing one.

Reply:
It is just draft but if you like the idea I'll change both moes/moesS thermostats for using this one with proper namings
(made pr as draft)